### PR TITLE
Bluesky Vector Scan Conversion

### PR DESCRIPTION
**Pull request by mskinner5278**

### CODE REVIEW

**Review by JunAishima — CHANGES_REQUESTED** [review]
I don't think the flyer should be protocol-dependent (at least not between standard and vector). can you think of a way that this dependency can be removed so that it can be used for both types of experiments?

**Comment by JunAishima** [comment]
also, please clean up the linting issues

File: nyxtools/flyer_eiger2.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ def configure_vector(self, **kwargs):
         scan_width = kwargs["scan_width"]
         exposure_ms = kwargs["exposure_period_per_image"] * 1.0e3
         num_images = kwargs["num_images"]
-        x_mm = (kwargs["x_start_um"] / 1000, kwargs["x_start_um"] / 1000)
-        y_mm = (kwargs["y_start_um"] / 1000, kwargs["y_start_um"] / 1000)
-        z_mm = (kwargs["z_start_um"] / 1000, kwargs["z_start_um"] / 1000)
+        x_mm = (kwargs["x_start_um"] / 1000, kwargs["x_end_um"] /1000)
```
Comment (RobertSchaffer1) [inline nyxtools/flyer_eiger2.py:56]: Add back in the space after the / sign.

**Review by JunAishima — APPROVED** [review]
looks good

**Pull request merged**